### PR TITLE
feat(#548): http support test in spring-fat

### LINKED ISSUE #548: Add `http` support to the `spring-fat` integration test
The `spring-fat` integration test starts a simple Spring Application with a single bean and this bean prints some logs to the stdout. If stdout contains required logs, we can say that test passes successfully. You can read more about the integration test right [here](https://github.com/objectionary/jeo-maven-plugin/tree/master/src/it/spring-fat).

Now we want make this test more "realistic". It would be nice to add `http` support to the application. The possible scenario might be:

1. Start Spring Application with some HTTP server (Spring Web?)
2. Make one http request to this server
3. Check the response status and content

Of course, we need to make `jeo-maven-plugin:disassemble` and `jeo-maven-plugin:assemble` before running the testing script described above.

### PR DESCRIPTION
**Pull request by h1alexbel**
In this pull I've added http support via Spring Web test into `spring-fat` it.

closes #548
History:
- **feat(#548): http test**
- **feat(#548): all params**
- **feat(#548): passes**
- **feat(#548): clean for qulice**

---

## PR-Codex overview
This PR focuses on refactoring the `Receptionist` class to utilize Spring's web capabilities, replacing the console output with a REST endpoint. It also updates the application configuration and adds integration tests for the new functionality.

### Detailed summary
- Added `invoker.goals` property to `invoker.properties`.
- Replaced `@Component` with `@RestController` in `Receptionist`.
- Created a new `/hello` endpoint in `Receptionist`.
- Removed the `CommandLineRunner` implementation in `Application`.
- Changed `spring-boot-starter` to `spring-boot-starter-web` in `pom.xml`.
- Added `spring-boot-starter-test` dependency for testing.
- Introduced integration tests in `ApplicationTest` for the `/hello` endpoint.
- Added logging configuration in `spring.factories`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by h1alexbel** [comment]
@volodya-lombrozo take a look, please

File: src/it/spring-fat/pom.xml [hunk 1]
```diff
@@ -109,6 +117,16 @@ SOFTWARE.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```
Comment (volodya-lombrozo) [inline src/it/spring-fat/pom.xml:121]: @h1alexbel Does it necessary? Can this test work without specifying `-parameters` argument?

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@h1alexbel Thank you for the contribution! 

Just wanted to suggest to add one more assertion to the `verify.groovy` script. For now it's not clear if we actually run this test or not.  It would be nice to see any prove of it.  
We already have some check:
```groovy
assert log.contains("Glad to see you, Fat Spring..."): "'Glad to see you, Fat Spring...' entry is not found"
```
but in your `junit` test you have the following statements which prints the same line into logs:

```java
final String expected = "Glad to see you, Fat Spring...";
System.out.printf("Expected %s%n", expected);
```
It means, even if test fails, the current check will show `true`.

**Comment by h1alexbel** [comment]
@volodya-lombrozo updated pull request, take a look, please

**Review by volodya-lombrozo — APPROVED** [review]
@h1alexbel Looks good to me. It’s a valuable addition to our work. Great job!

**Comment by volodya-lombrozo** [comment]
@rultor merge

**Pull request merged**